spa/src/components/common/Button/PublishButton/PublishButton.tsx

```diff
@@ -69,6 +70,8 @@ function PublishButton({ className }: PublishButtonProps) {
     handleOpen: handleOpenUnpublishModal
   } = useModal();
   const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+  const [slugError, setSlugError] = useState<string[] | null>(null);
```

Comment:
A cleaner way of doing this is:

```suggestion
  const [slugError, setSlugError] = useState<string[]>();
```